Missing script build --dev

When I run "npm run android:dev" the following output comes up:

```
> Ionic2-2LinkDossierApp@0.0.1 android:dev E:\Emil\Documents\FHNW\Semester 5\IP5\TEST\2LinkDossierApp
> bnr android:dev

running better-npm-run in E:\Emil\Documents\FHNW\Semester 5\IP5\TEST\2LinkDossierApp
Executing script: android:dev

to be executed: "ionic run android --livereload"

Running 'run:before' npm script before run
npm
 ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build --dev"

npm ERR! node v6.7.0
npm ERR! npm  v3.10.3



npm
ERR! missing script: build --dev
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>



npm ERR! Please include the following file with any support request:
npm ERR!     E:\Emil\Documents\FHNW\Semester 5\IP5\TEST\2LinkDossierApp\npm-debug.log

Caught exception:
 undefined

Mind letting us know? https://github.com/driftyco/ionic-cli/issues
```

--- a/package.json
+++ b/package.json
@@ -32,6 +32,7 @@
         "deploy:before": "build",
         "build:before": "build",
         "run:before": "build --dev",
+        "build --dev": "npm run build --dev",
         "ios:dev": "bnr ios:dev",
         "ios:release": "bnr ios:release",
         "ios:release:ci": "bnr ios:release:ci",

